ps_checkout.php

```diff
@@ -105,16 +106,19 @@ class Ps_checkout extends PaymentModule
         'PS_CHECKOUT_LIABILITY_SHIFT_REQ' => '1',
         'PS_CHECKOUT_DISPLAY_LOGO_PRODUCT' => '1',
         'PS_CHECKOUT_DISPLAY_LOGO_CART' => '1',
+        'PS_CHECKOUT_HOSTED_FIELDS_CONTINGENCIES' => 'SCA_WHEN_REQUIRED',
+        'PS_CHECKOUT_DOMAIN_REGISTERED_SANDBOX' => false,
+        'PS_CHECKOUT_DOMAIN_REGISTERED_LIVE' => false,
     ];
 
     public $confirmUninstall;
     public $bootstrap;
 
     // Needed in order to retrieve the module version easier (in api call headers) than instanciate
     // the module each time to get the version
-    const VERSION = '7.3.6.3';
+    const VERSION = '7.4.2.2';
```

Comment:
It can be 7.4.2.0, actually, as you prefer.

Reply:
Same here